spec/jobs/send_webhook_job_spec.rb

```diff
@@ -89,7 +91,7 @@
     end
 
     it 'calls the webhook event service' do
-      described_class.perform_now(
+      subject.perform_now(
```

Comment:
RSpec/NamedSubject: Name your test subject if you need to reference it explicitly. (https://rspec.rubystyle.guide/#use-subject, https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/NamedSubject)